web_src/js/jquery.js

```diff
@@ -1,3 +1,76 @@
 import $ from 'jquery';
 
 window.$ = window.jQuery = $;
+
+function getComputedStyleProperty(el, prop) {
+  const cs = el ? window.getComputedStyle(el) : null;
+  return cs ? cs[prop] : null;
+}
+
+const defaultDisplayMap = {};
+function getDefaultDisplay(el) {
+  let display = defaultDisplayMap[el.nodeName];
+  if (display) return display;
+
+  const temp = el.ownerDocument.body.appendChild(el.ownerDocument.createElement(el.nodeName));
+  display = getComputedStyleProperty(el, 'display');
```

Comment:
Don't `getComputedStyleProperty` in a hot code path like this, it's abhorrently slow.

Reply:
That's how jQuery does, and there  is a cache. So far so good, no more  problem

Reply:
jQuery legacy code is a very bad example to follow. Even if you cache it, I think the property accessor on the returned `style` object will still be slow regardless. But in any case, this whole solution is not something we should pursue.

Reply:
This PR doesn't make anything worse. It's a must at the moment.

You can fine  tune anything after the code gets refactored correctly.

Reply:
I added more comments:


> to remove the calls to "getComputedStyle", it requires that there is no element with "display: none" in its own CSS classes
>
> for example: `.popup { display: none; ... } <div class="popup">` should be rewritten to `.popup { ... } <div class="popup gt-hidden">`